DDDEastAnglia/Controllers/SessionController.cs

```diff
@@ -172,7 +178,7 @@ public ActionResult Delete(string userName, int id = 0)
             }
 
             var userProfile = userProfileRepository.GetUserProfileByUserName(session.SpeakerUserName);
-            var displayModel = CreateDisplayModel(session, userProfile);
+            var displayModel = CreateDisplayModel(session, userProfile, true);
```

Comment:
hard coded true for showSpeaker here, rather than a toggle.
Assume this is intended?

Reply:
👍 Good spot.